### PR TITLE
Enable 4.4 test cluster for odo test

### PR DESCRIPTION
**Pull request by amitkrout**
Enabling 4.4 test cluster for odo testing in OpenShift CI

### CODE REVIEW

**Comment by amitkrout** [comment]
```
 Running odo with args [odo component create nodejs:8 --project ifmgxpstlu --context /tmp/759313305]
[odo]  ✗  image stream nodejs with tag 8 not found in openshift and ifmgxpstlu namespaces 
```
This need to be fixed in odo test repo.

**Comment by mohammedzee1000** [comment]
oh, yea this is going to be a problem, we should probably update setup script to add correct IS as per test or we need to find a way to refer to IS without version tags

**Comment by amitkrout** [comment]
ping @prietyc123 (reviewer) @girishramnani (approver)

File: ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml [hunk 1]
```diff
@@ -353,3 +353,102 @@ presubmits:
...
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --kubeconfig=/etc/apici/kubeconfig
+        - --lease-server-password-file=/etc/boskos/password
+        - --lease-server-username=ci
+        - --lease-server=https://boskos-ci.svc.ci.openshift.org
+        - --secret-dir=/tmp/secret
```
Comment (prietyc123) [inline ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml:381]: @amitkrout Are we going to keep it as a future clearance of issue : https://github.com/openshift/odo/issues/2537?

Comment (amitkrout) [reply ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml:381]: This is created by the job generator. Later i will clean up those. BTW keeping those configuration is not hurting us atm.  

Anyway good catch. I will create a separate issue/pr to cleanup those from all cluster configuration.

**Comment by amitkrout** [comment]
> /approve

@girishramnani Seems you cannot approve. Let's wait for @kadel or @cdrage then.

**Pull request merged**